app/javascript/dashboard/routes/dashboard/settings/inbox/components/BusinessDay.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <div class="day-wrap">
+    <div class="checkbox-wrap">
+      <input
+        v-model="isDayEnabled"
+        name="enable-day"
+        class="enable-day"
+        type="checkbox"
+        :title="$t('INBOX_MGMT.BUSINESS_HOURS.DAY.ENABLE')"
+      />
+    </div>
+    <div class="day">
+      <span>{{ dayName }}</span>
+    </div>
+    <div v-if="isDayEnabled" class="hours-select-wrap">
+      <div class="hours-range">
+        <multiselect
+          v-model="fromTime"
+          :options="timeSlots"
+          deselect-label=""
+          select-label=""
+          selected-label=""
+          placeholder="Choose"
+          :allow-empty="false"
+        />
+        <div class="separator-icon">
+          <i class="ion-minus-round" />
+        </div>
+        <multiselect
+          v-model="toTime"
+          :options="timeSlots"
+          deselect-label=""
+          select-label=""
+          selected-label=""
+          placeholder="Choose"
+          :allow-empty="false"
+        />
+      </div>
+      <div v-if="hasError" class="date-error">
+        <span class="error">{{
+          $t('INBOX_MGMT.BUSINESS_HOURS.DAY.VALIDATION_ERROR')
+        }}</span>
+      </div>
+    </div>
+    <div v-else class="day-unavailable">
+      <span>
+        {{ $t('INBOX_MGMT.BUSINESS_HOURS.DAY.UNAVAILABLE') }}
+      </span>
+    </div>
+    <div>
+      <span v-if="isDayEnabled && !hasError" class="label">
+        {{ totalHours }} {{ $t('INBOX_MGMT.BUSINESS_HOURS.DAY.HOURS') }}
+      </span>
+    </div>
+  </div>
+</template>
+
+<script>
+import parse from 'date-fns/parse';
+import differenceInMinutes from 'date-fns/differenceInMinutes';
+import { generateTimeSlots } from '../helpers/businessHour';
+
+const timeSlots = generateTimeSlots(30);
+
+export default {
+  components: {},
+  props: {
+    dayName: {
+      type: String,
+      default: '',
+      required: true,
+    },
+    timeSlot: {
+      type: Object,
+      default: () => ({
+        from: '',
+        to: '',
+      }),
+    },
+  },
+  computed: {
+    timeSlots() {
```

Comment:
This is an unnecessary method.

Reply:
The constant `timeSlots` needs to be accessed in template